### DISCUSSION TITLE
brew install

### DISCUSSION THREAD

**Discussion by ghost** [contour-terminal/contour#1281] [General]
This looks awesome and is there a brew install for macos. Thank you. Best regards, Gaurav

**Comment by christianparpart** [comment]
> This looks awesome and is there a brew install for macos. Thank you. Best regards, Gaurav

You're welcome, @sablokgaurav.

I was assuming that you just wanted to give some feedback, and thus, I've converted this issue into a discussion, if you don't mind.

In case of any critique or suggestion on how to further improve, please feel free to contact us (through ticket system) or simply creating a discussion thread for anything else.

**Reply by ghost** [reply]
I dont mind anything, this would be really awesome and since this is a terminal with the gpu support so it would be awesome to have the same. I dont code in C anymore, otherwise would have added bindings :D. if you want me to write some bindings for the shell integration so as to make it more versatile then lets have a discussion and you will have it in no time.

**Reply by christianparpart** [reply]
Well, we have a ticket on shell integrations (#163), where the shell integration for Bash is still missing in order to close this ticket. In case you're using bash, we'd actually really welcome some help from outside, to continue moving forward :)

Other than that, we're currently in process of making sure we publish a next release (0.4.0), and tidy up all the odd bits still missing. But it'll take some time until we've tacked it all (e.g. MacOS + Qt 6 don't play nice for CPack generated .DMG files just yet. I'll need some time to research on how to fix this).

**Reply by ghost** [reply]
i use zsh and use two terminals one warp and one tabby and will start using this one also. I will have a look and will let you know. I will look at the code.

**Reply by ghost** [reply]
I looked at the code but i have never used the Qt6 as i never went for the graphical interface, I was more into the data analysis and everything.

**Reply by Yaraslaut** [reply]
No worries, at the moment brew repo does not contain contour package, we provide macos  package at release page [latest release](https://github.com/contour-terminal/contour/releases/tag/v0.3.12.262) , but at @christianparpart  mensioned we are planning to make new release soon, so you can give it a try with previous release and then update to newer version or build from source.

**Reply by ghost** [reply]
Thank you i didnt know that there is a binary release version. Actually was configuring a different interface using the glib and broke my kernel and so fixing it. : D . All configuration gone and have to reinstall it and with this phone internet, everything is so difficult :D

**Comment by ghost** [comment]
<img width="1676" alt="Screenshot 2023-10-16 at 10 32 48 PM" src="https://github.com/contour-terminal/contour/assets/19677953/4d616614-ebfe-4080-8362-bbd6612e0aa9">
see i give it a try but the font size is too small and it want to increase it as my vision is not too good, so cant see just few inches away. Is there any way that we can increase this.

**Reply by Yaraslaut** [reply]
Great :) 
You can customize `contour` to your taste in the config file, which by default is in `$HOME/.config/contour/contour.yml`, you can find a lot of stuff there, and font size is naturally one of the parameters

**Reply by ghost** [reply]
Thank you and i wanted to reply yesterday but my kernel broke down after installation and this needs a lot of customization such as while pressing the crtl tab, it doesnt open new window and everything. So i uninstalled it and fixed my kernel back. Thank you for this and i close this one.
